src/java/fr/paris/lutece/portal/business/user/attribute/AttributeRadioButton.java

```diff
@@ -237,6 +238,7 @@ public List<AdminUserField> getUserFieldsData( HttpServletRequest request, Admin
     public List<AdminUserField> getUserFieldsData( String [ ] strValues, AdminUser user )
     {
         List<AdminUserField> listUserFields = new ArrayList<>( );
+        AttributeFieldService attributeFieldService = CDI.current( ).select( AttributeFieldService.class ).get( );
```

Comment:
Do not use static calls; inject the service as a property since this class is a CDI bean.
@Inject 
private AttributeFieldService attributeFieldService;